data/npc/lib/npcsystem/npchandler.lua

```diff
@@ -76,29 +76,29 @@ if NpcHandler == nil then
 		topic = nil,
 		messages = {
 			-- These are the default replies of all npcs. They can/should be changed individually for each npc.
-			[MESSAGE_GREET]		= "Greetings, |PLAYERNAME|.",
-			[MESSAGE_FAREWELL] 	= "Good bye, |PLAYERNAME|.",
-			[MESSAGE_BUY] 		= "Do you want to buy |ITEMCOUNT| |ITEMNAME| for |TOTALCOST| gold coins?",
-			[MESSAGE_ONBUY]		= "Here you are.",
-			[MESSAGE_BOUGHT] 	= "Bought |ITEMCOUNT|x |ITEMNAME| for |TOTALCOST| gold.",
-			[MESSAGE_SELL] 		= "Do you want to sell |ITEMCOUNT| |ITEMNAME| for |TOTALCOST| gold coins?",
-			[MESSAGE_ONSELL] 	= "Here you are, |TOTALCOST| gold.",
-			[MESSAGE_SOLD]	 	= "Sold |ITEMCOUNT|x |ITEMNAME| for |TOTALCOST| gold.",
-			[MESSAGE_MISSINGMONEY]	= "You don't have enough money.",
-			[MESSAGE_NEEDMONEY] 	= "You don't have enough money.",
-			[MESSAGE_MISSINGITEM] 	= "You don't have so many.",
-			[MESSAGE_NEEDITEM]	= "You do not have this object.",
+			[MESSAGE_GREET] = "Greetings, |PLAYERNAME|.",
+			[MESSAGE_FAREWELL] = "Good bye, |PLAYERNAME|.",
+			[MESSAGE_BUY] = "Do you want to buy |ITEMCOUNT| |ITEMNAME| for |TOTALCOST| gold coins?",
+			[MESSAGE_ONBUY] = "Here you are.",
+			[MESSAGE_BOUGHT] = "Bought |ITEMCOUNT|x |ITEMNAME| for |TOTALCOST| gold.",
+			[MESSAGE_SELL] = "Do you want to sell |ITEMCOUNT| |ITEMNAME| for |TOTALCOST| gold coins?",
+			[MESSAGE_ONSELL] = "Here you are, |TOTALCOST| gold.",
+			[MESSAGE_SOLD] = "Sold |ITEMCOUNT|x |ITEMNAME| for |TOTALCOST| gold.",
+			[MESSAGE_MISSINGMONEY] = "You don't have enough money.",
+			[MESSAGE_NEEDMONEY] = "You don't have enough money.",
+			[MESSAGE_MISSINGITEM] = "You don't have so many.",
+			[MESSAGE_NEEDITEM] = "You do not have this object.",
 			[MESSAGE_NEEDSPACE]	= "You do not have enough capacity.",
```

Comment:
Fail haha, tought id spend a few mins but kept finding annoying things :p
Ill make sure to fix it tomorow!